content/docs/2.17/scalers/temporal.md

```diff
@@ -0,0 +1,108 @@
++++
+title = "Temporal"
+availability = "v2.17+"
+maintainer = "Community"
+description = "Scale applications based on Temporal task queue."
+go_file = "temporal_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `temporal` trigger that scales based on a temporal task queue.
+
+```yaml
+triggers:
+  - type: temporal
+    metadata:
+      namespace: default
+      taskQueue: "workflow_with_single_noop_activity:test"
+      targetQueueSize: "2"
+      activationTargetQueueSize: "0"
+      endpoint: temporal-frontend.temporal.svc.cluster.local:7233
+```
+
+**Parameter list:**
+
+- `endpoint` - This parameter specifies the URL of the Temporal gRPC service. You need to provide the service address in the format `<hostname>:<port>`. (Required)
+- `endpointFromEnv` - Defines the endpoint, similar to the `endpoint` parameter, but the value is read from an environment variable. (Optional)
+- `namespace` - The namespace of the temporal service. (Default:`default`, Optional)
+- `activationTargetQueueSize` - This sets the target value for activating the scaler. More information about activation thresholds can be found  [here](./../concepts/scaling-deployments.md#activating-and-scaling-thresholds). (Default: `0`, Optional)
+- `targetQueueSize` - Target value for queue length passed to the scaler. The scaler will cause the replicas to increase if the queue message count is greater than the target value per active replica. (Default: `5`, Optional)
+- `taskQueue` - This parameter specifies the task queue name. (Required)
+- `queueTypes` - Task Queue type, which can be either workflow or activity. The default type is workflow (Optional)
```

Comment:
```suggestion
- `queueTypes` - Task Queue type, which can be either `workflow` or `activity`. (Default: `workflow`, Optional)
```

Do we support only one value here? Then I don't we we should use a plural from (`types`), shoudln't it be `queueType` ?

Reply:
We support both. I will mention the same.